components/HomeHero/Strapline/types.ts

```diff
@@ -0,0 +1,8 @@
+type IconChoice = 'thumbs up' | 'thumbs down';
+
+type StraplineProps = {
+    icon: IconChoice;
```

Comment:
**Suggestion**

I don't see `IconChoice` being used anywhere else so we could just declare this one as 

```
type StraplineProps = {
  icon: 'thumps up' | 'thumbs down';
  ...
}
```

Reply:
It's used later in the HomePage type definition.